Runtime/Scripts/AudioSource.cs

```diff
@@ -69,7 +69,6 @@ public RtcAudioSource(AudioSource source, RtcAudioSourceType audioSourceType = R
             newAudioSource.Options.EchoCancellation = true;
             newAudioSource.Options.AutoGainControl = true;
             newAudioSource.Options.NoiseSuppression = true;
-            newAudioSource.EnableQueue = false;
```

Comment:
This line of code seems to need to be restored. This is a Unity-specific issue. Since there will be jitter when the Unity Audio Device captures audio, the queue needs to be closed inside ffi to avoid delay accumulation.

Reply:
This field was removed in https://github.com/livekit/rust-sdks/pull/425 - @theomonnom do you know if theres a different workaround we need to apply in Unity?

Reply:
ah, okay, theo has removed the silent frame insertion in ffi core, so this line of code can be removed